### ISSUE TITLE
Fix urlpatterns warning

### ISSUE THREAD

**Issue by iuriikomarov** [bfirsh/django-ordered-model#89]
```
lib/python3.5/site-packages/ordered_model/admin.py:32: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='{app}_{model}_order_down'.format(**self._get_model_info())),
```

**Comment by PavlinGergov** [comment]
Anything here?

**Comment by urda** [comment]
I thought this was fixed with issue #73 and PR #75.

However in issue #91 it looks like it's not been released yet. So I think we can consider this issue a duplicate and should refer to those that I have linked in this message.

**Comment by bfirsh** [comment]
Fixed in 1.2.0 :)
